docs/iov-name-service/validator/04-upgrade.md

```diff
@@ -0,0 +1,18 @@
+---
+id: upgrade
+title: Switching Testnet
+sidebar_label: Switching Testnet
```

Comment:
I created an addtionnal page about switching testnet so user is not overload with information not needed what he is starting to setup a full node. @davepuchyr